ReactiveCocoa/UIKit/iOS/UISearchBar.swift

```diff
@@ -39,7 +36,7 @@ extension Reactive where Base: UISearchBar {
 	/// - note: To observe text values that change on all editing events,
```

Comment:
The `important` comment for this signal and the `continuousTextValues` signal can be updated, now that `delegate`s are proxied.